css/style.css

```diff
@@ -327,3 +327,36 @@ div#tadd select { width: 220px }
 @media only screen and (max-width: 729px) {
   .mobile-search { display: none; }
 }
+
+/* Custom break point settings for responsiveness */
+
+/* X-Small devices (portrait phones, less than 576px) */
+/* No media query for `xs` since this is the default in Bootstrap */
+/* Custom rules for medium devices */
+.dlg-window {max-width: 95vw;}
+
+/* Small devices (landscape phones, 576px and up) */
+@media (min-width: 576px) { 
+  /* Custom rules for small devices */
+}
+
+/* Medium devices (tablets, 768px and up) */
+@media (min-width: 768px) { 
+  /* Custom rules for medium devices */
+  .dlg-window {max-width: 600px;}
+}
+
+/* Large devices (desktops, 992px and up) */
+@media (min-width: 992px) { 
+  /* Custom rules for large devices */
+}
+
+/* X-Large devices (large desktops, 1200px and up) */
+@media (min-width: 1200px) { 
+  /* Custom rules for x-large devices */
+}
+
+/* XX-Large devices (larger desktops, 1400px and up) */
+@media (min-width: 1400px) { 
+  /* Custom rules for xx-large devices */
+}
```

Comment:
Please test this custom values here and let me know what you think. They expand the size of the text boxes and dialog window, so the user can see more of the text being inputted on larger screens.

```
.dlg-window {max-width: 95vw;}

/* Medium devices (tablets, 768px and up) */
@media (min-width: 768px) { 
  /* Custom rules for medium devices */
  .dlg-window {min-width: 525px;}
  div#tadd .col-md-6 {min-width: 300px;}
  div#tadd .col-md-3 {max-width: 100px;}
}

/* Large devices (desktops, 992px and up) */
@media (min-width: 992px) { 
  /* Custom rules for large devices */
  .dlg-window {min-width: 675px;}
  div#tadd .col-md-6 {min-width: 450px;}
  div#tadd .col-md-3 {max-width: 100px;}
}
```

Reply:
I've done the test, and here's the screenshot for the `md` sized screen. 
![Screenshot 2024-06-22 173731](https://github.com/Novik/ruTorrent/assets/26022962/22187b4f-5ae9-46c8-8732-bbf3dcb0e504)

The four checkboxes below the directory input become unaligned with the directory input, as the `offset-md-3` uses the `margin-left` variable to implement the offset.

I think we can leave the `min-width` control on our dialog windows, to make better use of the wider display areas. However, on the columns inside a container, I think it's better to let Bootstrap handle the width distribution. For one thing, in this case, we can't have the maximum offset defined in CSS. For another, if we are to have nested columns in other dialog windows, it will require additional rules to exclude those nested columns. All in all, it will become harder to maintain in the future.

```CSS
/* Medium devices (tablets, 768px and up) */
@media (min-width: 768px) { 
  /* Custom rules for medium devices */
  .dlg-window {min-width: 525px;}  /* this line stays */
  div#tadd .col-md-6 {min-width: 300px;}  /* this line goes */
  div#tadd .col-md-3 {max-width: 100px;}  /* this line goes too */
}

/* Large devices (desktops, 992px and up) */
@media (min-width: 992px) { 
  /* Custom rules for large devices */
  .dlg-window {min-width: 675px;}  /* this line stays */
  div#tadd .col-md-6 {min-width: 450px;}  /* this line goes */
  div#tadd .col-md-3 {max-width: 100px;}  /* this line goes too */
}
```